src/components/tasks/card/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { FunctionComponent } from 'react';
+
+type pullRequest = {
+  title: string,
+  completionDate: string,
+  startedAt: string,
+  author: string,
+  profilePicture: string,
+  issueStatus: string
+}
+
+type Props = {
+  pullRequest: pullRequest
+}
+
+const Card: FunctionComponent<Props> = ({ pullRequest }) => {
+  const {
+    title,
+    completionDate,
+    startedAt,
+    author,
+    profilePicture,
+    issueStatus
+  } = pullRequest
+
+  return (
+    <>
+      <style jsx>{`
+        .card {
```

Comment:
NIT : I think we should move the css code to a separate file and import the file accordingly.
This will make it much more cleaner.
PS - I personally do not like to add css code this way